index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet"> 
+  <link rel="stylesheet" href="./css/style.css">
+  <title>My Portfolio - Squeleton</title>
+</head>
+<body>
+  <nav class="toolbar">
+    <nav class="welcome">
+      <h2>Welcome</h2>
+    </nav>
+    <nav class="dropbox">
+      <a href="#"><img id="drp-img" src="./images/dropbox.png" alt="dropbox"></a>
+    </nav>
+  </nav>
+  <nav class="headline">
```

Comment:
- [ ] Kindly use `<section></section>` instead of `<nav></nav>` and put it outside of the toolbar container (It's ok to have an entire container wrapping the page, but this appear to be a sub-container of the toolbar wrapper)